### PR TITLE
Separation of the EJBCA from the x509-identity-mgmt in a new container

### PR DESCRIPTION
**Pull request by thiagodpf**
The x509-identity-mgmt component now only has javascript code.
The configuration part of the EJBCA was separated into another
own container. This decision was made in order to scale the
javascript container.

The folder structure has also been changed. Now both
x509-identity-mgmt and EJBCA are located within the
"iam" (Identity and Access Management) directory.

No code files have been changed, just changed directories. 
The ejbca configuration scripts have occasional changes to maintain compatibility with the Nodejs application.

The only changed files that are worth seeing are the Dockerfile.
The original Dockerfile was removed and two new ones were created.

A file was also created to generate the image of the EJBCA on github actions.

A new example directory has been created containing a docker-compose.yml
that demonstrates the operation of the separate EJBCA and x509-identity-mgmt.

Note: the Health Check Verification has been removed from the Dockerfile, 
as it is the orchestrator's responsibility, in this case, it must be in the 
docker-compose.yml file or kubernetes.
See the docker-compose.yml in the example folder for more details.

* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

improvement

* **What is the current behavior?** (You can also link to an open issue here)

EJBCA and nodejs application within the same component (x509-identity-mgmt).

* **What is the new behavior (if this is a feature change)?**

A large component became two smaller. 
EJBCA started to have its own container, while x509-identity-mgmt was left with only the part in NodeJS.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

Yes, you will need to change the Docker-compose and Ansible repository.

* **Is there any issue related to this PR? (Link to an issue, e.g. #99999)** 

No ticket created yet

* **Other information**:

### CODE REVIEW

File: iam/x509-identity-mgmt/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+COPY --chown=10001:0 --from=base /usr/local/app /opt/x509-identity-mgmt/
+
+VOLUME ["/opt/tls/"]
+
+# Unhandled Promise Rejections is a newer feature that came up in Node.js 12.
+# Not handling promise rejections is an accepted practice in browsers, but in
+# servers it could be problematic because it can cause memory leaks.
+# To avoid this, we must use the flag 'unhandled-rejections' in 'strict' mode
+# so that the application ends immediately with an uncaught exception.
+CMD ["node","--unhandled-rejections=strict", "index.js"]
```
Comment (rascaraficci) [inline iam/x509-identity-mgmt/Dockerfile:66]: :+1:

File: iam/examples/x509_ejbca/README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+Username: admin
+Password: +ihKMor2H24GikhRFNZFPCLP
+***************************************************
+* Once the P12 is downloaded, use the password to *
+* import it on browser.                           *
+***************************************************
+```
+
+The important thing about this output is the values in front of *Username*
+(**admin**) and *Password* (**+ihKMor2H24GikhRFNZFPCLP**). In your case, this
```
Comment (rascaraficci) [inline iam/examples/x509_ejbca/README.md:62]: these values

File: iam/examples/x509_ejbca/README.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
+# Docker-compose example
```
Comment (rascaraficci) [inline iam/examples/x509_ejbca/README.md:1]: Nice documentation!

**Review by rascaraficci — APPROVED** [review]
Very good job!

File: iam/examples/x509_ejbca/README.md (hunk 2 above)
Comment (thiagodpf) [reply iam/examples/x509_ejbca/README.md:62]: done!

**Review by rascaraficci — APPROVED** [review]

**Review by mprevide — APPROVED** [review]

**Pull request merged**